website/content/api-docs/system/version-history.mdx

```diff
@@ -0,0 +1,54 @@
+---
+layout: api
+page_title: /sys/version-history - HTTP API
+description: The `/sys/version-history` endpoint is used to retrieve the version history of a Vault.
+---
+
+# `/sys/version-history`
+
+~> **NOTE**: Tracking Vault version history was added in Vault version 1.9.0. Entries for versions installed prior to version 1.9.0 will not be available via this API.
```

Comment:
```suggestion
~> **NOTE**: Tracking Vault version history was added in Vault version 1.9.0. Entries for versions installed prior to version 1.9.0 will not be available via this API.
This API was added in 1.10.0.
```

Although we don't normally say what version an endpoint was introduced in, I fear that if we don't in this case, people will read the above as "sys/version-history appeared in 1.9.0".

Reply:
I went ahead and added this. There are a handful of examples in our docs where we mention what version an endpoint was added. Like you said, it makes a lot of sense here given the callout of version tracking being introduced in 1.9.0.